diff -up layer-shell-qt-master/CMakeLists.txt.omv~ layer-shell-qt-master/CMakeLists.txt
--- layer-shell-qt-master/CMakeLists.txt.omv~	2023-05-20 22:28:50.398612489 +0200
+++ layer-shell-qt-master/CMakeLists.txt	2023-05-20 22:29:09.251774795 +0200
@@ -51,7 +51,7 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std
 ecm_setup_version(${PROJECT_VERSION} VARIABLE_PREFIX LAYERSHELLQT
                         VERSION_HEADER "${CMAKE_CURRENT_BINARY_DIR}/layershellqt_version.h"
                         PACKAGE_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/LayerShellQtConfigVersion.cmake"
-                        SOVERSION 5)
+                        SOVERSION 6)
 
 file(GLOB_RECURSE ALL_CLANG_FORMAT_SOURCE_FILES *.cpp *.h)
 kde_clang_format(${ALL_CLANG_FORMAT_SOURCE_FILES})
diff -up layer-shell-qt-master/src/CMakeLists.txt.omv~ layer-shell-qt-master/src/CMakeLists.txt
